src/routes/view_list_monitor/index.js

```diff
@@ -116,7 +110,8 @@ const ViewListMonitor = () => {
 							{val.success_rate_history.map((history, idx)=>(
 								<SuccessRate success={history.success} failed={history.failed} key={idx} /> 						
 							))}	
-							</div>											
+							</div>
+							<p>Last checked: {val.last_result?.execution_time}</p>													
```

Comment:
Seingatku format yg di execution_time masih blm readable, jdi perlu direformat dlu. Klo mau gampang, mungkin boleh tiru yg di error log list aja pake toLocateDateString

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
Oh ya, mungkin diupdate jg deskripsi PRnya dengan screenshot yg baru hehe titip ya fer, thank you 🙏

Reply:
ini kayaknya pake moment js aja fer

caranya 
- jalanin `npm install moment`
- lalu tulis `import moment from "moment"`
- lalu code yg val... diatas diganti jadi `{val.last_result ? moment(val.last_result.execution_time).fromNow() : "-"}`